src/components/OfflineWithFeedback.js

```diff
@@ -44,19 +44,17 @@ const propTypes = {
         PropTypes.object,
     ]),
 
-    /** Additional styles to add after local styles.  Applied to the error text portion */
-    errorStyle: PropTypes.oneOfType([
-        PropTypes.arrayOf(PropTypes.object),
-        PropTypes.object,
-    ]),
+    /** Additional style object for the error row */
+    errorRowStyles: PropTypes.arrayOf(PropTypes.object),
```

Comment:
Why did you make this stricter? Styles normally can be passed as an array of objects or as one object, why force to have an array always in this case?

Reply:
bleh, bad merge - not intentional